alf/experience_replayers/replay_buffer.py

```diff
@@ -70,19 +67,10 @@ def __init__(self,
                 of this value and the current maximum priority of all experiences.
             device (string): "cpu" or "cuda" where tensors are created.
             allow_multiprocess (bool): whether multiprocessing is supported.
-            her_k (float): proportion of hindsight relabeled experience.
+            keep_episodic_info (bool): index episode start and ending positions.
             step_type_field (string): path to the step_type field in exp nest.
                 This and the following fields are for hindsight relabeling.
-            achieved_goal_field (string): path to the achieved_goal field in
-                exp nest.
-            desired_goal_field (string): path to the desired_goal field in the
-                exp nest.
-            reward_field (string): path to the reward field in the exp nest.
-                are for hindsight experience replay.
-            reward_fn (callable): function to recompute reward based on
-                achieve_goal and desired_goal.  Default gives reward 0 when
-                L2 distance less than 0.05 and -1 otherwise, same as is done in
-                suite_robotics environments.
+            postprocess_exp_fn (callable): function to postprocess experience.
```

Comment:
Comment the expected arguments and return of this function.

Reply:
Done